charts/alfresco-repository/ci/kustomize/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+---
+namespace: alfresco
+secretGenerator:
+  - name: alfsecrets
+    literals:
+      - DATABASE_USERNAME=alfresco
+      - DATABASE_PASSWORD=alfresco
+generatorOptions:
+  disableNameSuffixHash: true
+
+helmChartInflationGenerator:
+  - chartName: postgresql
```

Comment:
in all the other charts we have migrated we have postgresql as optional dependency of the chart, stick to that

Reply:
Tracked i nhttps://alfresco.atlassian.net/browse/OPSEXP-2216 for now